server/models/User.js

```diff
@@ -42,6 +45,19 @@ const userSchema = new mongoose.Schema({
 	]
 });
 
+// Strips sensitive information when returning user
+userSchema.methods.toJSON = function () {
+	const user = this;
+	const userObject = user.toObject();
+
+	// Removing unnecessary information for the user to have
+	delete userObject.password;
+	delete userObject.tokens;
+	// delete userObject.avatar;
```

Comment:
remove?

Reply:
My comment doesn't explain this well, what this does is when for example a user is searching for someone in our database to send a message to, before we send them that user's data we will strip sensitive information such as their password and token. So when a user gets a response from our server, the only information they will have access to is anything but password and tokens.